baaas-decision-fleet-manager/src/main/java/org/kie/baaas/dfm/app/config/DecisionFleetManagerConfig.java

```diff
@@ -48,6 +48,10 @@
     @ConfigProperty(name = "baaas.dfm.s3.bucket.name")
     String bucketName;
 
+    @Inject
+    @ConfigProperty(name = "baaas.dfm.max.allowed.decisions")
+    long maxAllowedDecisions = -1; //disabled by default
```

Comment:
I think the assignment is redundant because you already have 
`baaas.dfm.max.allowed.decisions=${BAAAS_DFM_MAX_ALLOWED_DECISIONS:-1}`
in the `application.properties`